draft-ietf-httpbis-cache-latest.xml

```diff
@@ -1099,7 +1099,17 @@
 <t>
    Because unsafe request methods (<xref target="safe.methods"/>) such as PUT, POST or DELETE
    have the potential for changing state on the origin server, intervening
-   caches can use them to keep their contents up to date.
+   caches are required to invalidate stored responses to keep their contents up to date.
+   <x:dfn>Invalidate</x:dfn> means that the cache will either remove all
+   stored responses related to the effective request URI or will mark these
```

Comment:
I'd prefer "corresponding to" instead of "related to".

Reply:
I think the correct thing to say here is something like:

> Invalidate means that the cache will either remove all responses whose target URI matches the given URI, or...

because the actual requirements about what URI to invalidate are given below.

Reply:
okay